core/fixup.c

```diff
@@ -229,9 +229,9 @@ void fixup1zone(
   fixup_passive(i, j, k, pv, pv_prefloor);
 #endif
 
-#if ELECTRONS && EOS == EOS_TYPE_GAMMA
+#if ELECTRONS && EOS == EOS_TYPE_GAMMA_GASPRESS  
   // Reset entropy after floors
-  pv[KTOT] = EOS_Gamma_entropy_rho0_u(pv[RHO], pv[UU]);
+  pv[KTOT] = EOS_Gamma_Gaspress_entropy_rho0_u(pv[RHO], pv[UU]);
```

Comment:
I also wanted to check with you what this piece of code is doing. In the current modifications, I changed the name of the functions to the updated names specifying gas pressure. If this is going into evolving the simulation, should there also be another `if` statement for the radiation pressure dominated case as well?

Reply:
This does go into evolving the simulation, yes. But this code path will never be reached. The `ELECTRONS` machinery is backwards compatibility with `ebhlight`'s two-temperature gas for AGN disks. I would add another if case, but it's never going to come up.

Reply:
With `EOS_TYPE_GAMMA` having two sub-cases, this can go to `EOS_entropy_rho0_u` or something